server/boot/z-customer-accounts.js

```diff
@@ -0,0 +1,24 @@
+var async = require('async');
+
+module.exports = function(app) {
```

Comment:
Use async boot script since you're using async functions.

`module.exports = function(app, cb) {`